package.json

```
@@ -7,6 +7,7 @@
     "@types/node": "^12.11.1",
     "@types/react": "^16.9.9",
     "@types/react-dom": "^16.9.2",
+    "fhir-visualizers": "https://github.com/synthetichealth/fhir-visualizers#package",
```

I merged the PR on fhir-visualizers, so this can be updated. Other than this it looks good, but I'd like to wait until after Noranda's upcoming PR.
```suggestion
    "fhir-visualizers": "synthetichealth/fhir-visualizers",
```